app/Mail/CheckoutAssetMail.php

```diff
@@ -74,7 +74,7 @@ public function content(): Content
     {
         $this->item->load('assetstatus');
         $eula = method_exists($this->item, 'getEula') ? $this->item->getEula() : '';
-        $req_accept = method_exists($this->item, 'requireAcceptance') ? $this->item->requireAcceptance() : 0;
+        $req_accept = $this->requiresAcceptance();
```

Comment:
Are we sure we want to change this? I think this was there for checkout to location, other assets, etc.

Reply:
@snipe Double-check me but the old line is pulled as is into the private method so I can call it repeatedly in the `introductionLine()` method.